### ISSUE TITLE
GrowthMaps#dev?

### ISSUE THREAD

**Issue by nicolasmerino41** [cesaraustralia/GrowthMaps.jl#48]
Hey again,

Assuming you have other priorities before updating this package, is there a chance there's an available GrowthMaps#dev branch that is compatible with current versions of Rasters.jl, DD, etc? 

Thanks, 
Nico

**Comment by rafaqz** [comment]
There is no dev branch, I don't think much needs to change here except bumping some compat versions?

You are allowed to do that too, and make a PR :)

just `] dev GrowthMaps`, add the package versions you think we need to Project.jl, I guess switch Rasters to 0.11, but you can update everything else too.

Check you can install the latest Rasters, run the tests, push, PR. I can't imagine when I will personally have a spare hour to do that.

**Comment by nicolasmerino41** [comment]
Great, thank you! I'll try my best. Very new to Julia but happy to help :)

**Comment by rafaqz** [comment]
Ahh right, maybe thats a big ask then. You found a pretty obscure package to start with! 

I will try to bump these deps but I'm afraid the tests wont pass, even though the package will probably work.

**Comment by nicolasmerino41** [comment]
Yes, I've already done it and tests don't pass, I don't mind giving it a look and look for the incompatibilities til you find time.  It's good practice :)

**Comment by nicolasmerino41** [comment]
Hey there,

Don't know if a PR is the best way to tackle this so I first leave a comment here. I've checked the incompatibility issues and I've reached the following loop.
I've first bumped Reexport (and Rasters.jl too but that did not cause any issues):
`Reexport = "0.2, 2.0"`
But Reexport only allows up to StatsFuns@0.9.8 while Distributions.jl needs a minimum of StatsFuns@0.9.15. I've tried older versions of Distributions.jl and the oldest available one still requires over 0.9.15 so I kinda hit a wall there. I'm not very familiarised with the functionality of Reexport and StatsFuns so not sure how to tackle this. 

Let me know how you wanna tackle this and feel no rush, I'm interested in a couple of functionalities of the package but it's not a life or death thing.

Ps: Tests don't pass either now, but I think it's due to old Rasters.jl syntax so should be quite easy to fix. 

Good weekend,
Nico

**Comment by rafaqz** [comment]
Maybe try just with those packages in a new clean environment without GrowthMaps.jl? (Not your default environment with packages in it)

Those packages _should_ work together, if not you will need to fix their compat directly, it's not something I can fix here.

**Comment by nicolasmerino41** [comment]
True! You're right :)
